core/src/test/resources/strategies.properties

```diff
@@ -0,0 +1,3 @@
+smart.testing.strategy.new=org.arquillian.smart.testing:strategy-changed:0.0.11-SNAPSHOT
```

Comment:
I don't see this file used in some tests. Is it redundant?

Reply:
It was removed and moved to Maven extension,

Reply:
If it was removed, why this is added at the place: ` core/src/test/resources/strategies.properties`